runtime/tests/ecdsa.rs

```diff
@@ -12,140 +11,138 @@ use zerocopy::AsBytes;
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-            ],
-            sig_s: [
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
-            ],
-        },
-        // tcId: 176
-        // Signature with special case values r=1 and s=0
-        EcdsaTestCase {
-            msg: &[0x31, 0x32, 0x33, 0x34, 0x30, 0x30],
-            pub_x: pub_key_x,
-            pub_y: pub_key_y,
-            sig_r: [
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
-            ],
-            sig_s: [
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-            ],
-        },
-        // tcId: 193
-        // Signature with special case values r=n and s=1
-        EcdsaTestCase {
-            msg: &[0x31, 0x32, 0x33, 0x34, 0x30, 0x30],
-            pub_x: pub_key_x,
-            pub_y: pub_key_y,
-            sig_r: [
-                0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
-                0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xc7, 0x63, 0x4d, 0x81,
-                0xf4, 0x37, 0x2d, 0xdf, 0x58, 0x1a, 0x0d, 0xb2, 0x48, 0xb0, 0xa7, 0x7a, 0xec, 0xec,
-                0x19, 0x6a, 0xcc, 0xc5, 0x29, 0x73,
-            ],
-            sig_s: [
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-                0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
-            ],
-        },
-    ];
-
-    for c in cases {
-        // Stream to SHA ACC
-        model
-            .compute_sha512_acc_digest(c.msg, ShaAccMode::Sha384Stream)
+    #[allow(dead_code)]
```

Comment:
Not clear to me why this is dead code

Reply:
It silences the following warning:

warning: fields `id` and `comment` are never read
  --> runtime/tests/ecdsa.rs:25:9
   |
24 |     struct WycheproofResults {
   |            ----------------- fields in this struct
25 |         id: usize,
   |         ^^
26 |         comment: String,
   |         ^^^^^^^
   |
   = note: `WycheproofResults` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
   = note: `#[warn(dead_code)]` on by default

Reply:
Ah
> note: WycheproofResults has a derived impl for the trait Debug, but this is intentionally ignored during dead code analysis".

That's weird. Ok, fine to leave this then.